server/controllers/UserController.js

```diff
@@ -22,8 +22,9 @@ class UserController {
    */
   static signUpUser(req, res, next) {
     const {
-      email, username, password, bio, firstname, lastname
+      email, username, bio, firstname, lastname
```

Comment:
Each of the attribute should be on a single line

Reply:
I believe this is ok

Reply:
Thanks